service/rtc/sfu.go

```diff
@@ -592,13 +615,19 @@ func (s *Server) InitSession(cfg SessionConfig, closeCb func() error) error {
 				)
 
 				select {
-				case ss.tracksCh <- trackActionContext{action: trackActionAdd, track: pickRandom(outScreenTracks)}:
+				case ss.tracksCh <- trackActionContext{
+					action:        trackActionAdd,
+					localTrack:    pickRandom(outTracks),
```

Comment:
I don't think I fully understand why `pickRandom` is used here

Reply:
This is nothing new. Just applying the same optimization we use for screen tracks since they are still comparable in terms of type and rate. See https://github.com/mattermost/rtcd/pull/138

Reply:
ahhh, I forgot about that. Gotcha, thanks.